libs/documentation/src/lib/components/download/demos/basic/download-basic.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { SdsDialogService } from '@gsa-sam/components';
+import { SdsDownloadDisplayComponent } from './download-modal-display.component';
+
+@Component({
+  selector: 'gsa-sam-downloadbasic',
+  templateUrl: 'download-basic.component.html'
+})
+export class DownloadBasic implements OnInit {
+  @Input() message: string;
+  submitted = null;
+
+  constructor(private dialogService: SdsDialogService) {}
+
+  ngOnInit() {}
+
+  openModal() {
+    this.submitted = null;
+    this.dialogService
+      .open(SdsDownloadDisplayComponent)
```

Comment:
We don't need an intermediate component, we can simply pass data to the model which includes the formly variables.